### PR TITLE
Fix dt in envs

### PR DESCRIPTION
**Pull request by hunoutl**
In #15 dt was missing. simulta.py need it but was not in env config. Strange how everythings was working before ?!
I's not mandatory in AbstractEnv config class. Should it ?

### CODE REVIEW

**Comment by EtienneMenager** [comment]
Yes, it is strange. We have to add it in AbstractEnv.

**Comment by alxbilger** [comment]
@EtienneMenager would you like that this PR adds `dt` in `AbstractEnv`? Or it's ok to merge it like that?

File: test_env.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
         13:'trunkcup-v0',    
         } 
 
-num = 5
+num = 12
```
Comment (alxbilger) [inline test_env.py:47]: Is it part of the pull request?

**Comment by EtienneMenager** [comment]
I think it is better to integrate it now. There might be problems later...

File: test_env.py (hunk 1 above)
Comment (hunoutl) [reply test_env.py:47]: not important, used to switch env test.

**Comment by alxbilger** [comment]
Superseded by #19

**Pull request closed without merging**